contribs/gmf/src/layertree/SyncLayertreeMap.js

```diff
@@ -158,8 +158,19 @@ SyncLayertreeMap.prototype.updateLayerState_ = function (layer, treeCtrl) {
       if (gmfGroup.children === undefined && treeCtrl.getState() === 'on') {
         const gmfLayerWMS = /** @type {import('gmf/themes.js').GmfLayerWMS} */ (treeCtrl.node);
         names.push(gmfLayerWMS.layers);
-        const style = gmfLayerWMS.style !== undefined ? gmfLayerWMS.style : '';
-        styles.push(style);
+        if (gmfLayerWMS.style !== undefined) {
+          if (gmfLayerWMS.style.split(',').length > 1) {
+            if (gmfLayerWMS.style.split(',').length === gmfLayerWMS.layers.split(',').length) {
+              gmfLayerWMS.layers.split(',').forEach((style) => styles.push(style));
```

Comment:
In my understanding it should be:
```suggestion
              gmfLayerWMS.style.split(',').forEach((style) => styles.push(style));
```